ql/src/test/org/apache/hadoop/hive/ql/exec/vector/mapjoin/TestMapJoinOperator.java

```diff
@@ -1792,6 +1794,8 @@ private void executeTest(MapJoinTestDescription testDesc, MapJoinTestData testDa
     case FULL_OUTER:
       executeTestFullOuter(testDesc, testData, title);
       break;
+    case ANTI: //TODO
```

Comment:
Shall we open a ticket to track this? What is the main challenge here?

Reply:
https://issues.apache.org/jira/browse/HIVE-23904